orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/providers/aws/AmazonSecurityGroupUpserter.groovy

```diff
@@ -53,9 +53,16 @@ class AmazonSecurityGroupUpserter implements SecurityGroupUpserter, CloudProvide
         ).id
       }
 
+      // We used to get the `name` directly from the `context` which works
```

Comment:
Nitpick: this is a rather wordy comment and pasted a bunch of times.  maybe we can just put the details in a PR comment and then reference the PR on github? 
i.e., 
```
    // fall back to "name" for backwards compat -- https://github.com/spinnaker/orca/pull/2668
```

Reply:
👍

Reply:
yeah, i was trying to figure out a way to handle this better, I will leave the comment in one spot (AWS) and add a note to all the other providers to look at AWS

Reply:
nvm, looking at the next comment :)